variables.tf

```diff
@@ -356,3 +356,25 @@ variable "oidc_issuer_enabled" {
   type        = bool
   default     = false
 }
+
+variable "enable_key_vault_secrets_provider" {
```

Comment:
Can we rename this new added variable to `key_vault_secrets_provider_enabled` since the next variable's name is `secret_rotation_enabled`?